packages/browser/src/core/events/index.ts

```diff
@@ -205,6 +205,15 @@ export class EventFactory {
   }
 
   public normalize(event: SegmentEvent): SegmentEvent {
+    const anonymousIdOverride = event.options?.anonymousId
```

Comment:
maybe this is more readble?  
```
event.options?.anonymousId && this.user.anonymousId(event.options.anonymousId)
```
as `event.anonymousId = id` seems to be a noop and happens regardless of this line of code.

Reply:
Good catch!